packages/hoprd/src/api/v2.ts

```diff
@@ -247,6 +247,9 @@ export function setupWsApi(
       node.on('hopr:message', (msg: Uint8Array) => {
         socket.send(msg.toString())
       })
+      node.on(`hopr:message-acknowledged`, (ackChallenge: string) => {
```

Comment:
@nionis : would this be the right place to send acknowledgements to WS clients?

Reply:
Yes.
I wonder how this change will affect existing dApps.
We don't have a specification on the `format` of the incoming messages, we leave that fully up to the app, so I assume that the apps handle the "unparsable" messages already somehow.